sdk/eventhub/azure-eventhubs/azure/eventhub/_eventprocessor/local_checkpoint_store.py

```diff
@@ -0,0 +1,24 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from .sqlite3_checkpoint_store import Sqlite3CheckpointStore
+
+
+class InMemoryCheckpointStore(Sqlite3CheckpointStore):
+    """A checkpoint store that stores checkpoint and load balancer partition ownership data in memory.
+    This is for mock test only.
+
+    """
+    def __init__(self):
+        super(InMemoryCheckpointStore, self).__init__(db_filename=":memory:")
+
+
+class FileBasedCheckpointStore(Sqlite3CheckpointStore):
+    """A checkpoint store that stores checkpoint and load balancer partition ownership data in a file.
+    Do not use this
```

Comment:
Is there any way a user could accidentally see this?  (I see it's under an _eventprocessor path, so I'd assume no, but as always, paranoid double checking)

If so, and we're saying "Do not use" should this be "**_**FileBased..." (underscore)?

(Was reviewing pep8 + our conventions, the former seems to say that internal interfaces should be prefixed with a leading underscore, the latter says this MAY be omitted, if this is buried deep enough that a user would only find it by digging I wouldn't sweat it too much but figured I'd mention this.)

Reply:
This is going to removed from the package entirely before shipping.

Reply:
This will be completely removed, I would put that in a different PR.